hack/references/generate.go

```diff
@@ -92,7 +95,13 @@ func main() {
 			os.Exit(1)
 		}
 		specification := fmt.Sprintf("\n\n## Specification\n\n%s\n\n%s", specificationIntro, specificationContent)
-		refContent = title + description + specification + refContent
+
+		conflictWithAndMoreSection, err := generateConflictWithAndMore(capName)
```

Comment:
Do we need to write new code for each new feature in order to generate its corresponding document?

Reply:
No, run `hack/references/generate.go` will automatically generate all reference docs of those definitions by `vela workloads/traits`. As the generation tool doesn't cover `conflictsWith` and `How to use Rollout` section for Rollout Trait, I have to write new code for it.